From 2aaa5fded11c3599f2c29cb4abb6aa745dddd2f1 Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Thu, 2 Dec 2021 11:17:08 -0700
Subject: [PATCH] Compatibility to xtb 6.4.1

---
 xtb/qcschema/test_qcschema.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/xtb/qcschema/test_qcschema.py b/xtb/qcschema/test_qcschema.py
index 8cc0c2c..80dbe9a 100644
--- a/xtb/qcschema/test_qcschema.py
+++ b/xtb/qcschema/test_qcschema.py
@@ -63,7 +63,7 @@ def test_gfn2xtb_energy():
         }
     )
     dipole_moment = np.array(
-        [0.3345064021648074, -1.0700925215553294, -1.2299195418603437]
+        [0.33451151977076465, -1.0701017905608206, -1.229921234359929]
     )
 
     atomic_result = run_qcschema(atomic_input)
